src/app/Models/Setting.php

```diff
@@ -11,4 +11,31 @@ class Setting extends Model
 
     protected $table = 'settings';
     protected $fillable = ['value'];
+
+    public static function get($key)
+    {
+        $setting = new self();
+        $entry = $setting->where('key', $key)->first();
+
+        if (!$entry) {
+            return;
+        }
+
+        return $entry->value;
+    }
+
+    public static function set($key, $value = null)
+    {
+        $setting = new self();
+        $entry = $setting->where('key', $key)->first();
+
+        if (!$entry) {
+            return false;
+        }
+
+        $entry->update(['value' => $value]);
+        \Config::set('settings.'.$key, $value);
```

Comment:
So can this.

Reply:
True. Added check in https://github.com/Laravel-Backpack/Settings/commit/3064341962c9018df725ac17fb2defee24f7eef7